assets/scripts/script.js

```diff
@@ -53,7 +53,7 @@
           thumbnail: true,
```

Comment:
⬆️ Wondering whether we should upgrade the selector for the product gallery, too?  There is a more precise class now, which means that we also don't need to test for the wrapping context anymore.